resources/web/streaming_viewer/index.html

```diff
@@ -6,24 +6,24 @@
     <title>Streaming viewer</title>
     <link rel='stylesheet' href='https://code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css'/>
     <link type="text/css" rel="stylesheet" href='https://www.cyberbotics.com/jquery-ui-layout/1.4.4/layout-default.css' />
-    <link type="text/css" rel="stylesheet" href='https://www.cyberbotics.com/wwi/R2020b/wwi.css' />
+    <link type="text/css" rel="stylesheet" href='https://www.cyberbotics.com/wwi/R2020a/wwi.mjpeg.css' />
     <link type="text/css" rel="stylesheet" href='style.css' />
     <script src='https://code.jquery.com/jquery-3.1.1.min.js' integrity='sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8=' crossorigin='anonymous'></script>
     <script src='https://code.jquery.com/ui/1.12.1/jquery-ui.min.js' integrity='sha256-VazP97ZCwtekAsvgPBSUwPFKdrwD3unUfSGVYrahUqU=' crossorigin='anonymous'></script>
     <script src='https://www.cyberbotics.com/jquery-ui-layout/1.4.4/jquery.layout.js'></script>
     <script src='https://www.cyberbotics.com/jquery-dialogextend/2.0.4/jquery.dialogextend.min.js'></script>
     <script src='https://cdn.jsdelivr.net/ace/1.2.6/min/ace.js'></script>
-    <script src="https://www.cyberbotics.com/wwi/R2020b/webots.min.js"></script>
+    <script src="https://www.cyberbotics.com/wwi/R2020a/webots.mjpeg.js"></script>
     <script src="setup_viewer.js"></script>
   </head>
   <body>
     <div style="padding-left:10px">
       <h1>Webots streaming viewer</h1>
       <p>
         Connect to:
-        <input id="IPInput" type="text" value="localhost" />
+        <input id="IPInput" type="text" value="cyberbotics1.epfl.ch" />
```

Comment:
This are debug values in case you wanted to test the streaming directly from the streaming server.
But these changes have to be removed before merging.